tailcfg/tailcfg.go

```diff
@@ -535,15 +535,21 @@ type Service struct {
 // Tailscale host. Location is optional and only set if
 // explicitly declared by a node.
 type Location struct {
-	Country     string `json:",omitempty"` // User friendly country name, with proper capitalization, e.g "Canada"
-	CountryCode string `json:",omitempty"` // ISO 3166-1 alpha-2 in lower case, e.g "ca"
-	City        string `json:",omitempty"` // User friendly city name, with proper capitalization, e.g. "Squamish"
-	CityCode    string `json:",omitempty"`
-
-	// Priority determines the priority an exit node is given when the
-	// location data between two or more nodes is tied.
-	// A higher value indicates that the exit node is more preferable
-	// for use.
+	Country     string `json:",omitempty"` // User friendly country name, with proper capitalization (e.g "Canada")
```

Comment:
you lost the periods after g

but you can also just drop the latin and put the quoted string in parens and it's obvious enough that it's an gracious example